Selenium.Axe/Selenium.Axe/HtmlReport.cs

```diff
@@ -0,0 +1,249 @@
+﻿using HtmlAgilityPack;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Web;
+
+namespace Selenium.Axe
+{
+    public static class HtmlReport
+    {
+        public static void CreateReport(this IWebDriver webDriver, string destination)
```

Comment:
`CreateReport` is a pretty generic name for an extension method on `IWebDriver`; it would be easy for a different library to conflict with this. I think it would be less risky and more internally consistent for the API to work similarly to how `AxeBuilder.WithOutputFile` works, maybe add either a `WithHtmlReportFile(string destination)` or add an `OutputFileFormat` enum with "Html" and "AxeJson" options and a `WithOutputFile(string destination, OutputFileFormat format)` overload.

Reply:
Very good point, I had not thought about that.